src/components/common/StandardHeader.tsx

```diff
@@ -3,13 +3,17 @@ import { View } from 'react-native';
 
 interface HeaderContainerProps {
   children: ReactNode;
+  width?: string;
 }
 
-export default function HeaderContainer({ children }: HeaderContainerProps) {
+export default function HeaderContainer({
+  children,
+  width,
```

Comment:
Nit: If you'd like width to default to 90%, you should change line 11 to `width = '90%'` and remove the `|| '90%'` in line 16